cranelift/filetests/filetests/isa/x64/simd-bitwise-compile.clif

```diff
@@ -349,19 +350,21 @@ block0(v0: i8x16, v1: i32):
 ;   Entry block: 0
 ; Block 0:
 ;   (original IR block: block0)
-;   (instruction range: 0 .. 12)
+;   (instruction range: 0 .. 14)
 ;   Inst 0:   pushq   %rbp
 ;   Inst 1:   movq    %rsp, %rbp
-;   Inst 2:   movdqa  %xmm0, %xmm1
-;   Inst 3:   movdqa  %xmm1, %xmm0
-;   Inst 4:   punpcklbw %xmm0, %xmm0
-;   Inst 5:   psraw   $11, %xmm0
-;   Inst 6:   punpckhbw %xmm1, %xmm1
-;   Inst 7:   psraw   $11, %xmm1
-;   Inst 8:   packsswb %xmm1, %xmm0
-;   Inst 9:   movq    %rbp, %rsp
-;   Inst 10:   popq    %rbp
-;   Inst 11:   ret
+;   Inst 2:   movdqa  %xmm0, %xmm2
+;   Inst 3:   movdqa  %xmm2, %xmm1
+;   Inst 4:   punpcklbw %xmm2, %xmm1
+;   Inst 5:   psraw   $11, %xmm1
+;   Inst 6:   movdqa  %xmm2, %xmm0
+;   Inst 7:   punpckhbw %xmm2, %xmm0
+;   Inst 8:   psraw   $11, %xmm0
+;   Inst 9:   packsswb %xmm0, %xmm1
+;   Inst 10:   movdqa  %xmm1, %xmm0
```

Comment:
The register allocation here seems kinda poort where the `movdqa` here isn't present in the original code, and there's also an extra `movdqa` after the `psraw`. Would reordering the instructions in isle help at all?

I dunno how to keep track of "well regalloc2 will probably fix that" style things.

Reply:
I can poke at reordering the instructions, but I also think it isn't the biggest deal. We have a few tests that get a couple extra moves and also a few tests where we now elide a couple moves that we used to make. Overall I think it is pretty much a wash.